dpi/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "dpi"
```

Comment:
what's the point to name it dpi and not `winit-dpi`, since it's tailored towards the dpi needs of winit?

Reply:
Mostly because the name `dpi` was free, but also because the crate isn't really _fundamentally_ tied to Winit.